src/api/improve.rkt

```diff
@@ -33,7 +33,15 @@
 (define (run-improve input output #:threads [threads #f])
   (define seed (get-seed))
   (define tests (load-tests input))
-  (define results (get-test-results tests #:threads threads #:seed seed #:profile #f #:dir #f))
+  (start-job-server threads)
+  (define-values (ids)
+    (for/list ([test tests])
+      (define command
+        (create-job 'improve test #:seed seed #:pcontext #f #:profile? #f #:timeline-disabled? #f))
+      (start-job command)))
```

Comment:
Why turn off profile? I think we want it? Oh—wait, I think that flag probably just doesn't do anything any more.